edgecompute/examples/authentication/jwt-signing/README.md

```diff
@@ -0,0 +1,17 @@
+# JWT Signing
+
+_Keyword(s):_ jwt, authentication<br>
+
+This project provides an Akamai Edgeworker solution for signing JSON Web Tokens (JWTs) and attaching them to API requests. It includes an example of how to sign a JWT using HMAC-SHA256 and then add the JWT and an API key to the headers of an outgoing request.
+
+## Limitations
+
+-   As of now, EW do not support KMI to manage verification keys, hence these keys are fetched from property manager user defined variable which might not be a secure way. More details on user defined variables can be found [here](https://techdocs.akamai.com/property-mgr/docs/user-defined-vars).
```

Comment:
Rather than mentioning KMI, please provide more general advice: 

> Secrets are stored in [Property Manager variables](https://techdocs.akamai.com/property-mgr/docs/user-defined-vars). Property Manager variables should be marked as _hidden_ to prevent exposure with `akamai-x-get-extracted-values`. Depending on your security considerations, you may wish to store secrets elsewhere or use asymmetric keys.

Reply:
Done.